robot_modeling/DQ_SerialManipulatorDH.m

```diff
@@ -202,13 +160,152 @@
                 (d*cosine_of_half_alpha*cosine_of_half_theta)/2.0...
                 - (a*sine_of_half_alpha*sine_of_half_theta  )/2.0
                 ]);
+         end        
+        
+    end
+    
+    methods
+        function obj = DQ_SerialManipulatorDH(A,convention)
+            % These are initialized in the constructor of
+            % DQ_SerialManipulator
+            %obj.convention = convention;
+            %obj.n_links = size(A,2);
+              
+            obj = obj@DQ_SerialManipulator(size(A,2));
```

Comment:
Why do we need to use an argument in this constructor if the dimension of the configuration space can be inferred from the length of the matrix? Does it have to do with the way we must do in Python and C++?

Reply:
I am not sure if the argument in the constructor of DQ_SerialManipulator() is strictly required. I implemented it as it is in C++. @mmmarinho do you have any comments?